.env.example

```diff
@@ -0,0 +1,11 @@
+# Auth
+USERS_WHITELIST=<YOUR_GITHUB_USERNAME>
+GITHUB_TOKEN=<YOUR_GITHUB_TOKEN>
+
+# Clerk
+CLERK_SECRET_KEY=<YOUR_CLERK_SECRET_KEY>
+CLERK_WEBHOOK_SECRET=<YOUR_CLERK_WEBHOOK_SECRET>
+
+# Database
+DATABASE_URL=<YOUR_DATABASE_URL>
+DIRECT_URL=<YOUR_DIRECT_URL>
```

Comment:
Add the other env variables. Some of them are necessary to run correctly some features of our application.